Gemfile

```diff
@@ -172,6 +172,10 @@ gem "sprockets-rails", "3.4.2"
 gem "sprockets", "3.7.2"
 gem "uglifier"
 
+gem "eth", "~> 0.4"
+gem "rbnacl"
```

Comment:
no concerns from me for the addition of these dependencies. rbnacl calls down to libsodium which is a well known implementation that we already use